connectors/automation-anywhere/element-templates/automation-anywhere-connector.json

```diff
@@ -0,0 +1,692 @@
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "bearer"
+        ]
+      }
+    },
+    {
+      "label": "method for authentication",
+      "type": "Hidden",
+      "value": "post",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.method"
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "label": "Username",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.body.username"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "label": "Password",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.body.password"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "userNamePasswordAuth"
+      }
+    },
+    {
+      "label": "ApiKey",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.body.apiKey"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "userNameApiKeyAuth"
+      }
+    },
+    {
+      "label": "multipleLogin",
+      "group": "authentication",
+      "type": "Dropdown",
+      "value": "false",
+      "choices": [
+        {
+          "name": "TRUE",
+          "value": "true"
+        },
+        {
+          "name": "FALSE",
+          "value": "false"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.body.multipleLogin"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "userNamePasswordAuth"
+      }
+    },
+    {
+      "label": "Token",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "token"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "tokenAuth"
+      }
+    },
+    {
+      "label": "Token",
+      "group": "input",
+      "type": "Hidden",
+      "value": "={\"X-Authorization\":token}",
```

Comment:
Are you sure it should look like this? Usually the auth header `Authentication: Bearer {token}`